docs/data/toolpad/core/introduction/installation.md

```diff
@@ -72,3 +74,58 @@ yarn dev
 {{"component": "modules/components/DocsImage.tsx", "src": "/static/toolpad/docs/core/installation-1.png", "alt": "Toolpad Core entry point", "caption": "Starting with Toolpad Core", "zoom": true, "indent": 1 }}
 
 5. Installation is complete! Begin building your project by making edits to `(dashboard/page/page.tsx`. To understand how to leverage Toolpad Core to build dashboards quickly, [see the detailed tutorial](/toolpad/core/introduction/tutorial/).
+
+
+## Manual Installation
+
+Start by installing the required dependencies:
+
+<codeblock storageKey="package-manager">
+
+```bash npm
+npm install -S @toolpad/core next@latest react@latest react-dom@latest
+```
```

Comment:
Additionally, we also install following dependencies in a fresh app, should we also add them all in the same step?  
    "@mui/material": "next",
    "@mui/material-nextjs": "next",
    "@mui/icons-material": "next",
    "@emotion/react": "^11",
    "@emotion/styled": "^11",
    "@emotion/cache": "^11"